src/common/classes/tree.h

```diff
@@ -106,24 +93,24 @@ enum LocType { locEqual, locLess, locGreat, locGreatEqual, locLessEqual };
 // an indexed dynamic array without increase of algorithm calculation costs (this is one
 // more classical B+ tree feature). This is also not done to improve tree performance a little
 //
-template <typename Value, typename Key = Value, typename Allocator = MallocAllocator,
+template <typename Value, typename Key = Value, typename Allocator = Firebird::MemoryPool,
```

Comment:
Why not remove "Allocator" parameter at all?

Reply:
Good idea

Reply:
Done